rmbox/sof/uapi/ipc.h

```diff
@@ -0,0 +1,1020 @@
+/*
```

Comment:
Why are we duplicating this header ? @cujomalainey will be creating some git sub modules so we should have a local version of this header in the next day or so.

Reply:
Ok, I will update including headers, when the submodules are ready